### PR TITLE
update zh-trans content/zh/docs/reference/kubectl/kubectl.md

### PR DESCRIPTION
**Pull request by tanjunchen**
ref:https://github.com/k8smeetup/website-tasks/issues/3689
/cc @tengqm

### CODE REVIEW

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 1]
```diff
@@ -1,263 +1,620 @@
...
+
+    <tr>
+      <td colspan="2">--as string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      Username to impersonate for the operation
+      -->
+      指定用户执行操作
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
      以指定用户的身份执行操作
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 2]
```diff
@@ -1,263 +1,620 @@
...
+
+    <tr>
+      <td colspan="2">--as-group stringArray</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
+      -->
+      模拟操作的组，可以使用这个标识来指定多个组。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
      模拟指定的组来执行操作，可以使用这个标志来指定多个组。
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 3]
```diff
@@ -1,263 +1,620 @@
...
+
+    <tr>
+      <td colspan="2">--certificate-authority string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      Path to a cert file for the certificate authority
+      -->
+      用于进行认证授权的 cert 文件路径
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
      指向证书机构的 cert 文件路径
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 4]
```diff
@@ -1,263 +1,620 @@
...
+
+    <tr>
+      <td colspan="2">--log-backtrace-at traceLocation&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值: 0</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      when logging hits line file:N, emit a stack trace
+      -->
+      当日志长度超出规定的行数 N 时，忽略堆栈信息
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
      当日志机制运行到指定文件的指定行（file:N）时，打印调用堆栈信息
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 5]
```diff
@@ -1,263 +1,620 @@
...
+
+    <tr>
+      <td colspan="2">--log-file-max-size uint&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值: 1800</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      Defines the maximum size a log file can grow to. Unit is megabytes. If the value is 0, the maximum file size is unlimited.
+      -->
+      定义日志文件可以使用的最大值。单位为兆字节。如果值设置为 0，则表示日志文件大小不受限制。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
      定义日志文件的最大尺寸。单位为兆字节。如果值设置为 0，则表示日志文件大小不受限制。
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 6]
```diff
@@ -1,263 +1,620 @@
...
+
+    <tr>
+      <td colspan="2">--log-flush-frequency duration&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值: 5s</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      Maximum number of seconds between log flushes
+      -->
+      两次日志刷新操作之间的最大时间（秒）
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
      两次日志刷新操作之间的最长时间（秒）
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 7]
```diff
@@ -1,263 +1,620 @@
...
+
+    <tr>
+      <td colspan="2">--profile string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值: "none"</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      Name of profile to capture. One of (none|cpu|heap|goroutine|threadcreate|block|mutex)
+      -->
+      要捕获的信息的名称。是 (none|cpu|heap|goroutine|threadcreate|block|mutex) 其中之一。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
      要记录的性能指标的名称。可取 (none|cpu|heap|goroutine|threadcreate|block|mutex) 其中之一。
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 8]
```diff
@@ -1,263 +1,620 @@
...
+
+    <tr>
+      <td colspan="2">--profile-output string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值: "profile.pprof"</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      Name of the file to write the profile to
+      -->
+      要将捕获的信息写入的文件名
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
      用于转储所记录的性能信息的文件名
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 9]
```diff
@@ -1,263 +1,620 @@
...
+
+    <tr>
+      <td colspan="2">--request-timeout string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值: "0"</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      The length of time to wait before giving up on a single server request. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means don't timeout requests.
+      -->
+      丢弃一个简单服务请求前的等待时间，非零值需要包含相应时间单位(例如：1s, 2m, 3h)。零值则表示不做超时请求。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
      放弃单个服务器请求之前的等待时间，非零值需要包含相应时间单位（例如：1s、2m、3h）。零值则表示不做超时要求。
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 10]
```diff
@@ -1,263 +1,620 @@
...
+
+    <tr>
+      <td colspan="2">-s, --server string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      The address and port of the Kubernetes API server
+      -->
+      Kubernetes API server 的地址和端口
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
      Kubernetes API 服务器的地址和端口
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 11]
```diff
@@ -1,263 +1,620 @@
...
+
+    <tr>
+      <td colspan="2">--stderrthreshold severity&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值: 2</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      logs at or above this threshold go to stderr
+      -->
+      等于或高于此阈值的日志将输出到标准错误控制台
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
      等于或高于此阈值的日志将输出到标准错误输出（stderr）
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 12]
```diff
@@ -1,263 +1,620 @@
...
+
+    <tr>
+      <td colspan="2">--token string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      Bearer token for authentication to the API server
+      -->
+      用于 API server 进行身份认证的承载令牌
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
      用于对 API 服务器进行身份认证的持有者令牌
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 13]
```diff
@@ -1,263 +1,620 @@
...
+
+    <tr>
+      <td colspan="2">-v, --v Level</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      number for the log level verbosity
+      -->
+      指定输出日志的日志级别
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
      指定输出日志的日志详细级别
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 14]
```diff
@@ -1,263 +1,620 @@
...
+<!--
+* [kubectl convert](/docs/reference/generated/kubectl/kubectl-commands#convert)	 - Convert config files between different API versions
+* [kubectl cordon](/docs/reference/generated/kubectl/kubectl-commands#cordon)	 - Mark node as unschedulable
+* [kubectl cp](/docs/reference/generated/kubectl/kubectl-commands#cp)	 - Copy files and directories to and from containers.
+* [kubectl create](/docs/reference/generated/kubectl/kubectl-commands#create)	 - Create a resource from a file or from stdin.
+* [kubectl delete](/docs/reference/generated/kubectl/kubectl-commands#delete)	 - Delete resources by filenames, stdin, resources and names, or by resources and label selector
+-->
+* [kubectl convert](/docs/reference/generated/kubectl/kubectl-commands#convert)	 - 在不同的 API 版本之间转换配置文件
+* [kubectl cordon](/docs/reference/generated/kubectl/kubectl-commands#cordon)	 - 标记节点为不可调度的
+* [kubectl cp](/docs/reference/generated/kubectl/kubectl-commands#cp)	 - 将文件和路径拷入/拷出容器。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
* [kubectl cp](/docs/reference/generated/kubectl/kubectl-commands#cp)	 - 将文件和目录拷入/拷出容器。
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 15]
```diff
@@ -1,263 +1,620 @@
...
+* [kubectl scale](/docs/reference/generated/kubectl/kubectl-commands#scale)	 - Set a new size for a Deployment, ReplicaSet, Replication Controller, or Job
+* [kubectl set](/docs/reference/generated/kubectl/kubectl-commands#set)	 - Set specific features on objects
+* [kubectl taint](/docs/reference/generated/kubectl/kubectl-commands#taint)	 - Update the taints on one or more nodes
+* [kubectl top](/docs/reference/generated/kubectl/kubectl-commands#top)	 - Display Resource (CPU/Memory/Storage) usage.
+* [kubectl uncordon](/docs/reference/generated/kubectl/kubectl-commands#uncordon)	 - Mark node as schedulable
+* [kubectl version](/docs/reference/generated/kubectl/kubectl-commands#version)	 - Print the client and server version information
+* [kubectl wait](/docs/reference/generated/kubectl/kubectl-commands#wait)	 - Experimental: Wait for a specific condition on one or many resources.
+-->
+* [kubectl scale](/docs/reference/generated/kubectl/kubectl-commands#scale)	 - 为一个 Deployment、ReplicaSet、ReplicationController 或 Job 设置一个新的规模尺寸值
+* [kubectl set](/docs/reference/generated/kubectl/kubectl-commands#set)	 - 在资源对象设置特定的功能
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]: 为对象设置功能特性

**Comment by tanjunchen** [comment]
/cc @tengqm
done

**Pull request merged**